### PR TITLE
set locale for test, because test fails on german system

### PR DESCRIPTION
**Pull request by StefanSpieker**
On not english systems, the test will fail because the actual message is localized. So I set the locale before the test and reset it afterwards.

### Proposed changelog entries

* Internal

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [x] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [x] Appropriate autotests or explanation to why this change has no tests

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [x] There are at least 2 approvals for the pull request and no outstanding requests for change
- [x] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [x] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [x] Proper changelog labels are set so that the changelog can be generated automatically
- [x] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [x] If it would make sense to backport the change to LTS, a JIRA issue should exist and be labeled as `lts-candidate`

### CODE REVIEW

**Review by MarkEWaite — APPROVED** [review]

**Review by res0nance — APPROVED** [review]

**Review by alecharp — APPROVED** [review]

File: core/src/test/java/hudson/PluginWrapperTest.java [hunk 1]
```diff
@@ -48,13 +59,15 @@ public void optionalDependencyTest() {
...
     public void jenkinsCoreTooOld() throws Exception {
+
         PluginWrapper pw = pluginWrapper("fake").requiredCoreVersion("3.0").buildLoaded();
         try {
             pw.resolvePluginDependencies();
             fail();
         } catch (IOException ex) {
             assertContains(ex, "Failed to load: fake (42)", "Jenkins (3.0) or higher required");
         }
+        Locale.setDefault(loc);
```
Comment (rsandell) [inline core/src/test/java/hudson/PluginWrapperTest.java]: Why here? `after()` will run anyways right?

Comment (StefanSpieker) [reply core/src/test/java/hudson/PluginWrapperTest.java]: Yes, you are right. I forgot to remove it after moving it to the after().

**Comment by timja** [comment]
This pull-request is ready to merge, no need to wait any longer as its a simple test fix.

**Comment by MarkEWaite** [comment]
I agree with @timja .  It is ready to merge and has passed all the checklist items.

**Pull request merged**